Mathlib/Algebra/Polynomial/Taylor.lean

```diff
@@ -125,4 +125,11 @@ theorem sum_taylor_eq {R} [CommRing R] (f : R[X]) (r : R) :
   rw [← comp_eq_sum_left, sub_eq_add_neg, ← C_neg, ← taylor_apply, taylor_taylor, neg_add_self,
     taylor_zero]
 
+theorem eval_add_of_sq_eq_zero {A} [CommSemiring A] (p : Polynomial A) (x y : A) (hy : y ^ 2 = 0) :
```

Comment:
`y` can be implicit here.

Reply:
It needs to be explicit in my application. Do you prefer making `y` implicit here and using `(y := _)` at the application?

Reply:
If you need explicit then no problem leaving as it is. Thanks!